srcpkgs/png++-devel/template

```diff
@@ -0,0 +1,30 @@
+# Template file for 'png++-devel'
+pkgname=png++-devel
+version=0.2.9
+revision=1
+noarch=yes
+_pkgname=png++
```

Comment:
Maybe you want `libpng++` instead of this dance? Or even just `png++`.

Reply:
I saw the issue here.  The package name of the `png++-devel` is supposed to be `png++`.  I'm fixing it right now.